rootfs/api/migrations/0012_auto_20160810_0315.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should probably try to split this into 2 migrations, the stuff that's coming from all those other models and then yours.

Maybe make a separate PR for that, or i can